src/routes/devRoute.js

```diff
@@ -31,6 +32,7 @@ if (env !== 'production') {
   router.post('/update-user-score-phase2', auth.dev, updateUserScorePhase2);
   router.post('/remove-activity', auth.dev, removeActivity);
   router.post('/news-process', auth.dev, newsProcessDevProcess);
+  router.post('/trigger-topic-auto-message', triggerTopicAutoMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/trigger-topic-auto-message` is missing authentication middleware. This could potentially allow unauthorized users to trigger the auto message feature, which might lead to misuse or spamming. Please ensure that appropriate authentication middleware is applied to this route.
```diff
- router.post('/trigger-topic-auto-message', triggerTopicAutoMessage);
+ router.post('/trigger-topic-auto-message', auth.dev, triggerTopicAutoMessage);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->